keel-web/src/test/kotlin/com/netflix/spinnaker/keel/rest/DeliveryConfigControllerTests.kt

```diff
@@ -246,6 +253,12 @@ internal class DeliveryConfigControllerTests : JUnit5Minutests {
       ).forEach { (contentType, payload) ->
         derivedContext<ResultActions>("persisting a delivery config as $contentType") {
           fixture {
+            every {
+              repository.upsertDeliveryConfig(any<SubmittedDeliveryConfig>())
+            } answers {
+              firstArg<SubmittedDeliveryConfig>().toDeliveryConfig()
+            }
```

Comment:
Doing stub setup here is actually kind of a revelation to me. It seems obvious that you can, but I've always tried to do that in `before` which inevitably happens _after_ `fixture`, making it impossible to have a fixture like this which is the result of some action.

Reply:
TIL!